tests/src/Unit/ControllerTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+use Drupal\social_api\Controller\SocialApiController;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Drupal\social_api\Plugin\NetworkManager;
+use Drupal\Core\Cache\CacheBackendInterface;
+use Drupal\Core\Extension\ModuleHandlerInterface;
+use Drupal\Tests\UnitTestCase;
+
+/**
+ * Defines Controller.
+ *
+ * @Annotation
+ */
+class ControllerTest extends UnitTestCase {
```

Comment:
Use the same name as in the source code, but add Test at the end (`SocialApiControllerTest`)